docs/src/main/asciidoc/datasource.adoc

```diff
@@ -436,69 +421,77 @@ AgroalDataSource usersDataSource;
-is present and metrics for the Agroal extension are enabled. If you want to disable metrics for a particular data source,
-this can be done by setting `quarkus.datasource.jdbc.enable-metrics` to `false` (or `quarkus.datasource.<datasource name>.jdbc.enable-metrics` for a named datasource).
-This disables collecting the metrics as well as exposing them in the `/q/metrics` endpoint,
-because it does not make sense to expose metrics if the mechanism to collect them is disabled.
+To disable metrics for a particular data source,
+set `quarkus.datasource.jdbc.enable-metrics` to `false`, or apply `quarkus.datasource.<datasource name>.jdbc.enable-metrics` for a named datasource.
+This disables collecting the metrics and exposing them in the `/q/metrics` endpoint if the mechanism to collect them is disabled.
 
-Conversely, setting `quarkus.datasource.jdbc.enable-metrics` to `true` (or `quarkus.datasource.<datasource name>.jdbc.enable-metrics` for a named datasource) explicitly can be used to enable collection of metrics even if
-a metrics extension is not in use.
+Conversely, setting `quarkus.datasource.jdbc.enable-metrics` to `true`, or `quarkus.datasource.<datasource name>.jdbc.enable-metrics` for a named datasource explicitly enables metrics collection even if a metrics extension is not in use.
 This can be useful if you need to access the collected metrics programmatically.
-They are available after calling `dataSource.getMetrics()` on an injected `AgroalDataSource` instance. If collection of metrics is disabled
-for this data source, all values will be zero.
+They are available after calling `dataSource.getMetrics()` on an injected `AgroalDataSource` instance.
 
-== Narayana Transaction Manager integration
+If the metrics collection for this datasource is disabled, all values result to zero.
 
-If the Narayana JTA extension is also available, integration is automatic.
+=== Narayana transaction manager integration
 
-You can override this by setting the `transactions` configuration property - see the <<configuration-reference, Configuration Reference>> below.
+Integration is automatic if the Narayana JTA extension is also available.
 
-=== Named Datasources
+You can override this by setting the `transactions` configuration property.
+See the <<configuration-reference, Configuration Reference>> section below.
 
-When using Dev Services the default datasource will always be created, but to specify a named datasource you need to have
-at least one build time property so Quarkus knows to create the datasource. You will usually either specify
-the `db-kind` property, or explicitly enable Dev Services: `quarkus.datasource."name".devservices.enabled=true`.
+==== Named datasources
 
-[[in-memory-databases]]
-== Testing with in-memory databases
+When using Dev Services, the default datasource will always be created, but to specify a named datasource, you need to have at least one build time property so Quarkus can detect how to create the datasource.
```

Comment:
This seems unrelated to Narayana, and feels like a repeat from a previous note ("NOTE: Even when only one database extension is installed [...]").

I'd say this should be removed?

Reply:
Any opinion here? The section "Named datasources" within "Narayana transaction manager integration" really looks out of place and repeats what's already being said elsewhere.